test/metabase/db/schema_migrations_test.clj

```diff
@@ -9,37 +9,27 @@
 
   See `metabase.db.schema-migrations-test.impl` for the implementation of this functionality."
```

Comment:
removed all the migration tests for migration < 45 because they are no longer part of the main migration flow.

I could make it work and revert the deletion if someone think there is a good reason to keep those tests around.